artifact/gem5art/artifact/__init__.py

```diff
@@ -1,11 +1,5 @@
 """This is the gem5 artifact package"""
 
-from gem5art.artifact._artifactdb import ArtifactDB
-
-def getDBConnection() -> ArtifactDB:
-    """Returns the database connection
-
-    Eventually, this should likely read from a config file to get the database
-    information. However, for now, we'll use mongodb defaults
-    """
-    return ArtifactDB()
+from .artifact import Artifact
+from .artifact import getByName, getDiskImages, getLinuxBinaries, getgem5Binaries
```

Comment:
So, If I am not mistaken, we are only interested in having fewer artifacts (in import path) for those functions that will be used by the user to inspect the database?

Reply:
Yeah, the external interface functions. I was hoping I could use `__all__` in artifact.py, but that wasn't working for me as I expected.

Looking into it a bit more, we should probably specify an `__all__` in this file. I haven't read closely, but that seems to be what PEP 0008 is implying: https://www.python.org/dev/peps/pep-0008/#public-and-internal-interfaces